api-tester/src/main/java/com/revenuecat/apitester/kotlin/OfferingsAPI.kt

```diff
@@ -0,0 +1,14 @@
+package com.revenuecat.apitester.kotlin
+
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.Offerings
+
+@Suppress("unused", "UNUSED_VARIABLE")
+private class OfferingsAPI {
+    fun check(offerings: Offerings) {
+        val current: Offering? = offerings.current
+        val all: Map<String, Offering> = offerings.all
+        val o1: Offering? = offerings.getOffering("")
+        val o2: Offering? = offerings[""]
```

Comment:
Feel free to ignore but this function can also use `with(offerings)`. Which would require to do do `this[""]` in this line.

Reply:
I wasn't sure how to do that so I skipped it. Now I know how this works, defining `this` in that scope 😅 thanks, done!